draft-ietf-ccwg-bbr.md

```diff
@@ -754,6 +754,18 @@ protocols, and these implementations have been used in production
 for a large volume of Internet traffic. An open source implementation of
 BBR is also available for DCCP {{RFC4340}}  {{draft-romo-iccrg-ccid5}}.
 
+## ECN
+
+This experimental version of BBR does not specify a specific response to
+either Classic {{RFC3168}} or L4S {{RFC9330}} style ECN. However, if
+BBR negotiates either type of ECN, the congestion controller response
```

Comment:
Strictly speaking, BBR itself would not do the ECN negotiation itself in some trnsports, and there may be future transports that do not require negotiating ECN support; they may specify it as mandatory. AFAICT the constraint is with the ECN code point. So for precision I'd suggest replacing "BBR negotiates either type of ECN" with "the transport connection claims ECN support by marking  packets using either the ECT(0) or ECT(1) code point".

Reply:
```suggestion
the transport connection claims ECN support by marking packets using either the ECT(0) or ECT(1) code point, the congestion controller response
```